packages/esm-appointments-app/src/routes.json

```diff
@@ -39,6 +39,66 @@
       "name": "todays-appointments-dashboard",
       "slot": "todays-appointment-slot",
       "component": "homeAppointments"
+    },
+    {
+      "name": "expected-appointments-panel",
+      "slot": "scheduled-appointments-panels-slot",
+      "component": "appointmentsByStatus",
+      "meta": {
+        "title": "expected",
+        "status": "Scheduled",
+        "showForPastDate": false,
+        "showForToday": true,
+        "showForFutureDate": true
+      }
+    },
```

Comment:
Here's where I wired in the "default" extensions to display on the page.  We can debate what the actual defaults should be, but perhaps first discuss the way I set this up for configuration.  

We basically support the following meta parameters to configure the tabs as well as the AppointmentByStatus component:
* title - title (message code) to pass into the ApointmentsByStatus widget and to display in the tab header
* status - status (one of the valid Appointmnent Statuses) to pass into the AppointmentByStatus widget
* showForPastDate: whether this tab should be displayed for past dates
* showForToday: whether this tab should be displayed for current date
* showForFutureDate: whether this tab should be displayed for future dates